v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/changestream/DataChangeRecordTypeConvertor.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.changestream;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.cloud.ByteArray;
+import com.google.cloud.Date;
+import com.google.cloud.Timestamp;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.regex.Pattern;
+import org.apache.commons.lang3.math.NumberUtils;
+
+/**
+ * Utility class with methods which converts text fields in change events represented by JSONObject
+ * to Cloud Spanner types. The field types come from spanner change streams hence the data formats
+ * are going to the the same.
+ */
+public class DataChangeRecordTypeConvertor {
```

Comment:
This class looks like a duplicate, can we not use ChangeEventTypeConvertor from spanner-common